editor/src/clj/editor/code/view.clj

```diff
@@ -3583,14 +3536,15 @@
           lsp)
         goto-line-bar (setup-goto-line-bar! (ui/load-fxml "goto-line.fxml") view-node)
         find-bar (setup-find-bar! (ui/load-fxml "find.fxml") view-node)
-        replace-bar (setup-replace-bar! (ui/load-fxml "replace.fxml") view-node)
+        replace-bar (setup-replace-bar! (ui/load-fxml "replace.fxml") view-node editable)
         repainter (ui/->timer "repaint-code-editor-view"
                               (fn [_ elapsed-time _]
                                 (when (and (.isSelected tab) (not (ui/ui-disabled?)))
-                                  (repaint-view! view-node elapsed-time {:cursor-visible true}))))
+                                  (repaint-view! view-node elapsed-time {:cursor-visible true :editable editable}))))
         dispose-hover! (create-hover! view-node canvas tab)
         dispose-breakpoint-editor! (create-breakpoint-editor! view-node canvas tab)
         context-env {:clipboard (Clipboard/getSystemClipboard)
+                     :editable editable
```

Comment:
Putting the `editable` boolean in the UI `context` for the JavaFx `Controls` enables us to access it in the `(active? [editable] editable)` handlers.